examples/ecs/ecs_guide.rs

```diff
@@ -324,20 +329,17 @@ fn main() {
         )
         .add_system_to_stage(MyStage::BeforeRound, new_round_system)
         .add_system_to_stage(MyStage::BeforeRound, new_player_system)
+        .add_system_to_stage(MyStage::BeforeRound, exclusive_player_system.at_start())
+        // We can ensure that game_over system runs after score_check_system using explicit ordering
+        // constraints First, we label the system we want to refer to using `.label`
+        // Then, we use either `.before` or `.after` to describe the order we want the relationship
         .add_system_to_stage(
-            MyStage::BeforeRound,
-            // Systems which take `&mut World` as an argument must call `.exclusive_system()`.
-            // The following will not compile.
-            //.add_system_to_stage(MyStage::BeforeRound, exclusive_player_system)
-            exclusive_player_system.exclusive_system(),
+            MyStage::AfterRound,
+            score_check_system.label(MyLabels::ScoreCheck),
         )
-        .add_system_to_stage(MyStage::AfterRound, score_check_system)
         .add_system_to_stage(
-            // We can ensure that `game_over_system` runs after `score_check_system` using explicit ordering
-            // To do this we use either `.before` or `.after` to describe the order we want the relationship
-            // Since we are using `after`, `game_over_system` runs after `score_check_system`
             MyStage::AfterRound,
-            game_over_system.after(score_check_system),
+            game_over_system.after(MyLabels::ScoreCheck),
```

Comment:
This looks like a merge conflict mistake?

Reply:
Or have you not yet ported the auto-label stuff?

Reply:
probably